mlflow/R/mlflow/tests/testthat/test-model-mleap.R

```diff
@@ -5,8 +5,9 @@ library(mleap)
 for (i in 0:4){
     tryCatch(
         expr = {
-            host <- sprintf("local:%s", 8881 + i)
-            sc <- sparklyr::spark_connect(master = host, version = "2.4.5")
+            config <- spark_config()
```

Comment:
We might need `sparklyr::` here:

```
config <- sparklyr::spark_config()
```

Reply:
https://github.com/mlflow/mlflow/runs/4307358473?check_suite_focus=true#step:14:566

```
<simpleError in spark_config(): could not find function "spark_config">
<simpleError in spark_config(): could not find function "spark_config">
<simpleError in spark_config(): could not find function "spark_config">
<simpleError in spark_config(): could not find function "spark_config">
```

Reply:
yep :) re-running the test suite with referencing spark_config from sparklyr